src/core/builders/vite/plugins/devHtmlPrerender.ts

```diff
@@ -61,6 +61,17 @@ export function devHtmlPrerender(
               const abs = resolve(dirname(id), src);
               const pathname = relative(config.root, abs);
               element.setAttribute(attr, `${server.origin}/${pathname}`);
+            } else {
+              const [alias, path] = Object.entries(config.alias).find(([key]) =>
+                src.startsWith(key),
+              )!;
+
+              const baseUrl = `${server.origin}/${relative(config.root, path)}`;
```

Comment:
Looks really good, about what I was thinking. You'll need to call `normailzePath` on the relative path to make it work on Windows, otherwise the URL would look like `http://localhost:5173\assets\main.css`.

Reply:
Interesting. I guess I should add `normalize` from `node:path` on all the paths. No matter if they are absolute/relative or aliased?